src/input_graph.rs

```diff
@@ -206,6 +202,17 @@ impl InputGraph {
         g.freeze();
         g
     }
+
+    /// Writes the input graph to a text file, using the following format:
+    /// a <from> <to> <weight>
+    /// Mostly used for performance testing.
```

Comment:
If we want to keep DIMACS compatibility we would have to increment from/to by one, as the nodes must be 1-based. But then of course we also need to adjust the read method accordingly.